app/assets/stylesheets/ama_layout/layout_components/footer.scss

```diff
@@ -1,13 +1,37 @@
 // Sticky footer setup
-// See: https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/
-.off-canvas-wrapper{
-  display: flex;
-  flex-direction: column;
-  min-height: 100vh;
+$footer-push-small: 617px;
+$footer-push-medium: 432px;
+$footer-push-large: 329px;
+
+html, body{
```

Comment:
Can you try removing this CSS? If you still need it I dont think it should live in a footer file.